app/src/main/java/com/globant/example/mentorapp/domain/interactor/ListUsersInteractor.java

```diff
@@ -0,0 +1,16 @@
+package com.globant.example.mentorapp.domain.interactor;
+
+import com.globant.example.mentorapp.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.presentation.view.fragment.ListUsersInterface;
+
+import retrofit2.Retrofit;
+
+/**
+ * Created by oscar.amezquita on 9/06/2017.
+ */
+
+public interface ListUsersInteractor {
```

Comment:
Interactors should be based on Command pattern. The name of the class must be self-explanatory and only have one method called execute.